index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
           />
         </div>
       </section>
+      <!-- WORK SECTION -->
+      <section id="work">
+        <div class="card-works">
+          <img
+            src="./images/work/Tonic.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Tonic</div>
```

Comment:
According to the project requirements, the project titles should be wrapped inside a `hx` tag, kindly fix this.

<img width="346" alt="Screen Shot 2022-05-27 at 17 24 13" src="https://user-images.githubusercontent.com/82426682/170796124-dcb78aa9-3d81-4dee-88b3-beac5a2070dc.png">

You should fix all the titles, I'm marking just this one for brevity.